igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/logparser/PropertyParser.groovy

```diff
@@ -34,6 +38,17 @@ class PropertyParser {
                 log.info "${key}:${value}"
                 map.put(key,value)
             }
+            if (line =~ MAGIC_JSON_SEARCH_STRING) {
```

Comment:
Looks like you intend to have the magic string be the first non-whitespace substring in the line, would it be worth matching on that? `^\s*${MAGIC_JSON....`